nat-lab/tests/test_network_switch.py

```diff
@@ -320,14 +318,12 @@ async def test_mesh_network_switch_direct(
         async with run_async_contexts(
             [
                 derp_connected_future,
-                peers_connected_relay_future,
```

Comment:
Are we not getting relayed connection event anymore?

Reply:
Well, after the last change I guess I get them again, but they seems to be unnecessary - I got just Direct -> Direct endpoint change recently, which seems to be fine as well, but doesn't pass the test in its original form